Sources/Uno/OneTimePassword/Length.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Uno
+//
+//  Created by Gaetano Matonti on 06/09/21.
+//
+
+import Foundation
+
+public extension OneTimePassword {
+  /// The possible OTP codes length expressed in digits.
+  enum Length {
+    /// A six digits code.
+    case six
+    
+    /// A seven digits code.
+    case seven
+    
+    /// A eight digits code.
+    case eight
+    
+    // MARK: - Computed Properties
+    
+    /// The value representing the number of digits of the OTP code..
+    var value: Int {
+      switch self {
+        case .six:
+          return 6
+          
+        case .seven:
+          return 7
+          
+        case .eight:
+          return 8
+      }
+    }
+  }
```

Comment:
perchè non usare il rawvalue direttamente?

Reply:
in effetti si potrebbe. però non essendo `RawRepresentable` obbligo ad utilizzare la funzione con la gestione degli errori, anche se è solo interna al modulo.

Reply:
Potresti anche solo chiamarlo `rawValue`